integrations/axum/src/lib.rs

```diff
@@ -773,8 +773,15 @@ async fn generate_response(
     if let Some(status) = res_options.status {
         *res.status_mut() = status
     }
+    // Set the Content Type headers on all responses. This makes Firefox show the page source
+    // without complaining
+    let headers = res.headers_mut();
+    headers.insert(
```

Comment:
I think will overwrite a user-provided `CONTENT_TYPE` if there is one, right? I can't imagine why there would be one, to be fair—any page we're rendering is going to be HTML because of the boilerplate that gets inserted, and it's going to be UTF-8—but is it worth checking for any reason so we don't overwrite it?